Library/ViewModels/SettingsNotificationCellViewModel.swift

```diff
@@ -249,6 +250,9 @@ extension SettingsNotificationCellViewModel {
     case .friendBacksProject:
       return notificationType == .email
         ? UserAttribute.Notification.friendActivity : UserAttribute.Notification.mobileFriendActivity
+    case .commentReplyDigest:
```

Comment:
Should we also test this case?

Reply:
I don't think it's necessary because the notifications test that we have already are sufficient.